src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, partnerName, sex, isMarried } = person;
+
```

Comment:
add a helper variable
```suggestion
const partnerStatus = sex === 'm' ? 'wife' : 'husband';
```